gnoland/website/main_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+	"github.com/gotuna/gotuna/test/assert"
```

Comment:
Does anyone know why the linter didn't detect this issue in the CI and the PR is still marked as green?

Reply:
@grepsuzette could you run `make fmt` or fix the format of this file, please?

Reply:
Yes I configured my vim so it uses gopls and gofumpt, it reformatted it.